### PR TITLE
RЗС-145-audit 0 salt

### PR DESCRIPTION
**Pull request by SKonstantinS**

### CODE REVIEW

File: exchange-v2/test/v2/ExchangeV2.rarible.test.js [hunk 1]
```diff
@@ -62,33 +62,33 @@ contract("ExchangeV2, sellerFee + buyerFee =  6%,", accounts => {
 	beforeEach(resetState);
 
 	describe("on-chain orders", () => {
-        it("should create, update then cancel order", async () => {
```
Comment (disenotov) [inline exchange-v2/test/v2/ExchangeV2.rarible.test.js:65]: it would be better to fix formatting in separate PRs

Comment (SKonstantinS) [reply exchange-v2/test/v2/ExchangeV2.rarible.test.js:65]: Idea: fix format in test after all merge in RPC-143-Audit-OnChainOrders

**Comment by SKonstantinS** [comment]
OnChainOrders have to be refactoring, now close this PR
